test/test_ranker.py

```diff
@@ -1,42 +1,74 @@
 from nose.tools import eq_
+from nose import with_setup
 import math
+import os
+import json
 from tsg.config import DATA_DIR
 from tsg.ranker import get_dictionary_term_list, cosine_score_calc, calculate_query_term_weight, get_number_of_docs
 
-TEST_DICT_FILENAME = DATA_DIR + '/test/files/test_dict.dat'
+TEST_DICT_PATH = DATA_DIR + 'testdict.dat'
```

Comment:
It's the same as test_indexer. Those files are created and deleted in each test run so i believe it doesn't really matter where they are. If you want so then define in config.py a default testing location and we can replace it here.